packages/svelte/src/internal/client/dom/blocks/each.js

```diff
@@ -399,53 +385,31 @@ function reconcile(
 		for (i = 0; i < length; i += 1) {
 			value = array[i];
 			key = get_key(value, i);
-			item = items.get(key);
+			item = /** @type {EachItem} */ (items.get(key));
 
-			if (item !== undefined) {
-				item.a?.measure();
-				(to_animate ??= new Set()).add(item);
-			}
+			item.a?.measure();
+			(to_animate ??= new Set()).add(item);
 		}
 	}
 
 	for (i = 0; i < length; i += 1) {
 		value = array[i];
 		key = get_key(value, i);
 
-		item = items.get(key);
-
-		if (item === undefined) {
-			var pending = offscreen_items.get(key);
+		item = /** @type {EachItem} */ (items.get(key));
```

Comment:
I am running into a bug in my codebase due to this line making the assumption that item cannot be `undefined`.
I am not able to make a reproduction, should i open a bug report anyway ?

Reply:
Yes please, maybe someone else is able to make one, or you find a way, a non-minimal reproduction would also be fine in that case